apps/cli/src/commands/proxy.ts

```diff
@@ -250,9 +250,10 @@ const job = ({
         };
 
         ws.on("close", onClose);
-        ws.on("message", async (apduHex) => {
+        ws.on("message", async (data, isBinary) => {
```

Comment:
I think you can do `.toString()` on a string, and cover both cases

Reply:
Yup I know, I've added that just for everyone reading this code to understand why we're doing this and using the `isBinary` parameter.

Reply:
Also by answering Alexandre I also realised that data would be either `Buffer|ArrayBuffer` or `Buffer[]`, never a string anymore. Which means isBinary is true for `Buffer|ArrayBuffer` and false for `Buffer[]` (which is a weird API imo... 🤷 )

Reply:
Ok so looks like `.toString()` would also be fine on `Buffer[]`. I had no idea what it would do here.
<img width="952" alt="Screenshot 2022-06-16 at 14 07 11" src="https://user-images.githubusercontent.com/44363395/174066417-7b8d0446-0784-4889-8968-c19bd0f0804c.png">

Reply:
ah, we agree that the `isBinary` name is weird right ?! (i was thinking that i was dumb not understanding it)

Reply:
Yup but from that code [here](https://github.com/websockets/ws/blob/e173423c180dc1e4e6ee8938d9e4376a7a8b9757/lib/receiver.js#L491) we have the confirmation that data is `Buffer|ArrayBuffer|Buffer[]` so in our specific case, we could have the `.toString` in every case 🤔 

Anyway wdyt @juan-cortes, am I keeping the ternary for the sake of clarity or do we abstract that and parse as string no matter what since it doesn't change anything ?